### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by WJVincent**
Fixed Auth and Validation bugs

### CODE REVIEW

File: routes/users.js [hunk 1]
```diff
@@ -50,14 +50,18 @@ router.post(
       email,
       password,
     });
-
+    console.log("before");
     const validatorErrors = validationResult(req);
```
Comment (darabandev) [inline routes/users.js:54]: Remove console.log

File: routes/users.js [hunk 2]
```diff
@@ -50,14 +50,18 @@ router.post(
...
+    console.log(validatorErrors);
```
Comment (darabandev) [inline routes/users.js:55]: Remove console.log

File: routes/users.js [hunk 3]
```diff
@@ -50,14 +50,18 @@ router.post(
...
 
     if (validatorErrors.isEmpty()) {
+      console.log("after success");
```
Comment (darabandev) [inline routes/users.js:58]: Remove console.log

File: routes/users.js [hunk 4]
```diff
@@ -69,6 +73,7 @@ router.post(
         res.redirect("/");
       });
     } else {
+      console.log("after fail");
```
Comment (darabandev) [inline routes/users.js:76]: Remove console.log

**Review by darabandev — CHANGES_REQUESTED** [review]
Looks good!

File: routes/users.js (hunk 1 above)
Comment (WJVincent) [reply routes/users.js:54]: removed

File: routes/users.js (hunk 2 above)
Comment (WJVincent) [reply routes/users.js:55]: removed

File: routes/users.js (hunk 3 above)
Comment (WJVincent) [reply routes/users.js:58]: removed

File: routes/users.js (hunk 4 above)
Comment (WJVincent) [reply routes/users.js:76]: removed

**Review by darabandev — APPROVED** [review]

**Pull request merged**